### PR TITLE
update kubectl "$" format

### LINKED ISSUE #13222: update kubectl "$" formatting docs
A follow-up issue to track https://github.com/kubernetes/website/pull/12867#issuecomment-473471032

I'm ok with the change going in after the 1.14 release so it can be against master. Thanks!

/assign @Liujingfang1

### PR DESCRIPTION
**Pull request by Liujingfang1**
Fix #13222

### CODE REVIEW

**Comment by DanyC97** [comment]
/assign @DanyC97 

will try to review it today @Liujingfang1 , thanks for the PR

File: content/en/docs/tasks/administer-cluster/storage-object-in-use-protection.md [hunk 1]
```diff
@@ -303,10 +303,16 @@ task-pv-volume   1Gi          RWO            Delete           Terminating   defa
 
 ```shell
 kubectl delete pvc task-pv-claim
+```
```
Comment (DanyC97) [inline content/en/docs/tasks/administer-cluster/storage-object-in-use-protection.md]: please add _The output is similar to this:_ before the output

File: content/en/docs/tutorials/services/source-ip.md [hunk 2]
```diff
@@ -254,29 +303,42 @@ health check --->   node 1   node 2 <--- health check
...
 ```
-$ kubectl get pod -o wide -l run=source-ip-app
 NAME                            READY     STATUS    RESTARTS   AGE       IP             NODE
 source-ip-app-826191075-qehz4   1/1       Running   0          20h       10.180.1.136   kubernetes-minion-group-6jst
-
+```
+Curl the `/healthz` endpoint on different nodes.
+```console
 kubernetes-minion-group-6jst $ curl localhost:32122/healthz
+```
```
Comment (DanyC97) [inline content/en/docs/tutorials/services/source-ip.md]: please add _The output is similar to this:_ before the output

File: content/en/docs/tutorials/services/source-ip.md [hunk 3]
```diff
@@ -254,29 +303,42 @@ health check --->   node 1   node 2 <--- health check
...
+```
 1 Service Endpoints found
-
+```
+```console
 kubernetes-minion-group-jj1t $ curl localhost:32122/healthz
+```
```
Comment (DanyC97) [inline content/en/docs/tutorials/services/source-ip.md]: please add _The output is similar to this:_ before the output

**Comment by DanyC97** [comment]
@Liujingfang1 thank you for working on this PR. you mind resolving the conflict before i can take another look at ?

**Comment by Liujingfang1** [comment]
@DanyC97 I'll resolve the conflicts.

**Comment by DanyC97** [comment]
@Liujingfang1 thank you for making the changes. I'm not sure why those failures, i'll try to understand the reason behind it

**Comment by Rajakavitha1** [comment]
@Liujingfang1 could you please resolve the conflicts.

**Comment by DanyC97** [comment]
@tengqm much thanks !

**Pull request merged**